static/js/pages/profile.js

```diff
@@ -621,16 +648,17 @@ function unpinScore(id, rx) {
 }
 
 function pinButton(id, rx) {
-	return `<a href='#' class='new pinbutton' onclick='pinScore("${id}", ${rx})'><i class='pin icon'></i></a>`
+	return `<a href="#" class="new pinbutton" title="Pin Score" onclick='pinScore("${id}", ${rx})'><i class='pin icon'></i></a>`
 }
 
 function unpinButton(id, rx) {
-	return `<a href='#' class='new unpinbutton' onclick='unpinScore("${id}", ${rx})'><i class='pin icon'></i></a>`
+	return `<a href="#" class="new unpinbutton" title="Unpin Score" onclick='unpinScore("${id}", ${rx})'><i class='pin icon'></i></a>`
 }
 
 function downloadStar(id) {
-	return "<a href='/web/replays/" + id + "' class='new downloadstar'><i class='star icon'></i></a>";
+	return `<a href="/web/replays/${id}" title="Download Replay" class='new downloadstar'><i class='fa-solid fa-download icon'></i></a>`;
```

Comment:
this won’t work - js keeps it as an int which breaks autopilot. revert this pls

Reply:
ah i didn't see the quotes lol